docs/content/1.guide/7.plugins.md

```diff
@@ -27,6 +27,34 @@ export default defineNitroPlugin((nitroApp) => {
 })
 ```
 
+**Example:** Response Modification
+
+You can use plugins to register a hook that modifies the renderer response.
```

Comment:
We might link to the `renderer` option here and add a note it only works for when using the feature and Nuxt SSR not for all responses.